runtime-scripts/start-ovsdb-server.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright 2018 Cisco Systems Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+mkdir -p /var/run/openvswitch
+
+if ! lsmod | cut -d" " -f1 | grep -q openvswitch; then
+	echo "INFO: Loading kernel module: openvswitch"
+	modprobe openvswitch
+	sleep 2
+fi
```

Comment:
does the db need the module?

Reply:
@chrisplo: I don't think so. I'll update this shortly.